front/src/modules/timesStops/styles/_timesStopsDatasheet.scss

```
@@ -97,4 +97,14 @@
       --dsg-cell-disabled-background-color: var(--ambientB5);
     }
   }
+
+  .align-right-responsive {
+    text-align: right !important;
+  }
```

```suggestion
  .align-right-responsive {
    text-align: right;
    span {
      width: 100%;
    }
  }
```

You can remove the `!important` rule too :) (same below)